Jenkinsfile

```diff
@@ -7,7 +7,7 @@
 * Jenkins populates the patch version depending on the branch.
 */
 
-String VERSION = "2.28"
+String VERSION = "2.29"
```

Comment:
For major or minor version bumps, we also update the changelog in repo root

Reply:
whoops, updated :)